cmake/all.cmake

```diff
@@ -162,6 +166,11 @@ set(CATKIN_GLOBAL_LIBEXEC_DESTINATION lib)
 set(CATKIN_GLOBAL_PYTHON_DESTINATION ${PYTHON_INSTALL_DIR})
 set(CATKIN_GLOBAL_SHARE_DESTINATION share)
 
+# This was removed, but is required for Windows
```

Comment:
I believe that is my own comment, as my upgrade from Indigo to Jade tried to remove this addition.  I will test and ensure it is necessary.